src/main/assets/js/upload-manager/index.ts

```diff
@@ -17,14 +17,21 @@ import '@uppy/progress-bar/src/style.scss';
 const initUploadManager = (): void => {
   const url = DOCUMENT_MANAGER;
   const csrfToken = (getById('csrfToken') as HTMLInputElement)?.value;
+  const localeToken = (getById('localeToken') as HTMLInputElement)?.value;
   const csrfQuery = `?_csrf=${csrfToken}`;
   location.hash = '';
 
-  const chooseFilePhoto = 'Choose a file or take a photo';
+  let chooseFilePhoto;
+  if (localeToken === 'cy') {
+    chooseFilePhoto = 'Choose a file or take a photo (in welsh)';
+  } else {
+    chooseFilePhoto = 'Choose a file or take a photo';
+  }
 
   const uppy = new Uppy({
     restrictions: {
-      maxFileSize: 10485760,
+      maxFileSize: 2097152,
+      maxTotalFileSize: 10485760,
```

Comment:
what is the difference between `maxFileSize` and `maxTotalFileSize`

Reply:
maxFileSize is the max size of one file whereas maxTotalFileSize is the max size by all files combined